Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,74 @@
 import React from 'react';
+import Display from './components/DisplayComponents/CalculatorDisplay'
+import Button from './components/ButtonComponents/Button'
 import './App.css';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+let numbers = [9,8,7,6,5,4,3,2,1]
+let opers = ['÷','x','-','+','=']
```

Comment:
great job putting all of your numbers and operators into an array, makes it really easy to map though.